src/di.php

```diff
@@ -493,6 +493,15 @@
     }
 };
 
+$di['set_redirect_uri'] = function () use ($di) {
+    $url = $_GET['_url'] ?? ($_SERVER['PATH_INFO'] ?? '');
+    unset($_GET['_url']);
+
+    $redirectUri = str_replace('/admin', '', $url) . '?' . http_build_query($_GET);
```

Comment:
The admin area prefix can be changed, so we can't hard-code it to `/admin`, instead you should use the `ADMIN_PREFIX` constant